Github/service.js

```diff
@@ -66,9 +66,9 @@ const getCommits = commitObject => {
         headers: { 'User-Agent': 'request' },
       },
       (err, res, data) => {
-        if (err) {
-          console.log('Error:', err);
-          reject(err);
+        if (res.statusCode !== 200) {
+          console.log('Error:', res.statusMessage);
+          reject(new Error(`Unable to get repos.`));
```

Comment:
nitpick: unnecessary bactkticks (same in line 106)

Reply:
fixed